pkg/workflow/spec/fnlib/convert.go

```diff
@@ -0,0 +1,49 @@
+package fnlib
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/puppetlabs/nebula-sdk/pkg/workflow/spec/convert"
+	"github.com/puppetlabs/nebula-sdk/pkg/workflow/spec/fn"
+)
+
+var convertMarkdownDescriptor = fn.DescriptorFuncs{
+	DescriptionFunc: func() string { return "Converts a string in markdown format to another applicable syntax" },
+	PositionalInvokerFunc: func(args []interface{}) (fn.Invoker, error) {
+		fn := fn.InvokerFunc(func(ctx context.Context) (m interface{}, err error) {
+			if len(args) != 2 {
+				return nil, &fn.ArityError{Wanted: []int{2}, Variadic: true, Got: len(args)}
```

Comment:
This should be `Variadic: false` I think. Also, this would be a good function to also provide a `KeywordInvokerFunc` for -- `!fn.convertMarkdown {to: html, content: <markdown here>}` or something. Take a look at the `merge` function for an example of providing both.